variables.tf

```diff
@@ -186,3 +186,10 @@ variable "associate_public_ip_address" {
   type    = "string"
   default = "false"
 }
+
+// Additional Security Groups to assign to Instances in the ASG
+variable "additional_security_group_ids" {
+  type    = "list"
+  default = []
+  description = "Additional Security Groups to assign to Instances in the ASG"
```

Comment:
Can we remove the description here and move any detail that you want to provide into a full-english sentence here? When I release the next version of the module, I will be re-generating the docs with https://github.com/segmentio/terraform-docs, and the README will be re-generated with the comment from here.